### PR TITLE
remove some equipements types from the filters created using the map

### PR DESCRIPTION
**Pull request by Mathieu-Deharbe**
The filter creation on the map now uses directly the list of filterable equipment from commons-ui.

### CODE REVIEW

File: src/components/network/filter-creation-panel.tsx [hunk 1]
```diff
@@ -28,11 +29,8 @@ import {
 import { fetchElementsMetadata } from 'services/explore';
 import { UniqueNameInput } from 'components/dialogs/commons/unique-name-input';
 import { useSelector } from 'react-redux';
-import {
-    equipementTypeToLabel,
-    EQUIPMENT_TYPES,
-} from '../utils/equipment-types';
 import { UUID } from 'crypto';
+import { FormEquipment } from '@gridsuite/commons-ui/dist/components/filter/utils/filter-form-utils';
```
Comment (jamal-khey) [inline src/components/network/filter-creation-panel.tsx:33]: i think it not normal to include components from @gridsuite/commons-ui/**dist/**

**Review by jamal-khey — COMMENTED** [review]
imports to be changed

File: src/components/network/filter-creation-panel.tsx (hunk 1 above)
Comment (Mathieu-Deharbe) [reply src/components/network/filter-creation-panel.tsx]: Indeed. I simply switched it to `from '@gridsuite/commons-ui';`. Webstorm complains but it works fine.

I guess I miss some kind of config for the auto import.

**Review by jamal-khey — COMMENTED** [review]
Code and test Ok , but you have a error in the console:

```
@formatjs/intl Error MISSING_TRANSLATION] Missing message: "Hvdc" for locale "fr", using id as fallback.

    at new MissingTranslationError2
```

**Review by jamal-khey — APPROVED** [review]
Test and code OK
Console logs Ok

**Review by jamal-khey — APPROVED** [review]

**Pull request merged**